bin/setup_dot1x

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash -e
+
+ROOT=$(dirname $0)/..
+cd $ROOT
+
+sudo rm -rf inst/faucet/faucet_dot1x
+mkdir -p inst/faucet/faucet_dot1x
+
+sudo modprobe bonding
+
+cp misc/faucet_dot1x.yaml inst/faucet/faucet_dot1x/faucet.yaml
+
+docker rm -f faucet_dot1x || true
+docker rm -f daq-fr || true
+
+cmd/faux 1 xdhcp 
+cmd/faux 2 xdhcp 
+
+echo -e "Created faux"
+
+docker exec -ti daq-faux-1 ip addr flush faux-eth0
+docker exec -ti daq-faux-1 ifconfig faux-eth0 192.168.12.1/16
+docker exec -ti daq-faux-2 ip addr flush faux-eth0
+docker exec -ti daq-faux-2 ifconfig faux-eth0 192.168.12.2/16
+
+echo -e "set IPs for faux"
+
+docker run -d --privileged --name daq-fr --hostname daq-fr daq/networking
+docker exec -ti daq-fr ip addr flush eth0
+ip=$(ip -o -4 addr list docker0 | awk '{print $4}' | cut -d '/' -f1)2
+docker exec -ti daq-fr ifconfig eth0 $ip
+docker cp misc/clients.conf.freeradius daq-fr:/etc/freeradius/3.0/clients.conf
+docker cp misc/authorize.freeradius daq-fr:/etc/freeradius/3.0/mods-config/files/authorize
+docker exec -ti daq-fr freeradius -X &
```

Comment:
The config files as well? But those are specific to this setup.

Reply:
I would... they actually aren't very specific, and having them in there doesn't harm anything. Also, at some point in the future we'll want/need to test if a device is doing 802.1x correctly, and then it would be useful there too, I think... not critical.